terraform/ci.tfvars

```diff
@@ -10,3 +10,4 @@ support_email="govpaas-alerting-ci@digital.cabinet-office.gov.uk"
 # Enabled/disabled resources
 # Disable datadog_monitor.total_routes_drop resource
 datadog_monitor_total_routes_drop_enabled = 0
+pingdom_contact_ids=11089310,11190300
```

Comment:
It probably needs to be quoted.

Having said that, this looks like a good opportunity to update this to use an array variable.